ci-operator/step-registry/gs-baremetal/localnet-test/gs-baremetal-localnet-test-ref.yaml

```
@@ -0,0 +1,41 @@
+ref:
+  as: gs-baremetal-localnet-test
+  cli: latest
+  credentials:
+    - name: cluster-secrets-aws-interop-qe
+      namespace: test-credentials
+      mount_path: /secrets
+    - name: cnv-ci-brew-pull-secret
+      namespace: test-credentials
+      mount_path: /var/run/cnv-ci-brew-pull-secret
+    - name: openshift-virtualization-tests-credentials
+      namespace: test-credentials
+      mount_path: /bw
+  documentation: |-
+    Run localnet tests for Openshift Virtualization on OCP baremetal cluster configured for Goldman Sachs.
+  from: openshift-virtualization-tests
+  commands: gs-baremetal-localnet-test-commands.sh
+  timeout: 4h0m0s
+  grace_period: 2h
+  resources:
+    requests:
+      cpu: 100m
+      memory: 200Mi
+  env:
+  - name: BW_PATH
+    default: "/bw"
+    documentation: Bitwarden folder
+  - name: OCP_VERSION
+    default: "4.19"
+    documentation: |-
+      OpenShift Version
+  - name: BREW_IMAGE_REGISTRY_USERNAME
+    default: '|7820aea2-0d75-11e7-9259-28d244ea5a6d.hhav.f63e13'
+  - name: BREW_IMAGE_REGISTRY_TOKEN_PATH
+    default: /var/run/cnv-ci-brew-pull-secret/token
```

Minor suggestion: parameterize the storage class name for reuse
```suggestion
    default: /var/run/cnv-ci-brew-pull-secret/token
- name: DEFAULT_STORAGE_CLASS
    default: "ocs-storagecluster-ceph-rbd-virtualization"
```